test/statsCases/chunks/expected.txt

```diff
@@ -6,18 +6,44 @@ Time: Xms
 2.bundle.js  101 bytes       2  [emitted]  
 3.bundle.js  180 bytes       3  [emitted]  
 chunk    {0} bundle.js (main) 73 bytes [rendered]
+    > main [0] (webpack)/test/statsCases/chunks/index.js 
```

Comment:
The default should not change (else it would be a breaking change). Make sure these tests pass in their original form.